cli/Cargo.toml

```diff
@@ -155,6 +155,7 @@ http.workspace = true
 http-body-util.workspace = true
 hyper.workspace = true
 hyper-util.workspace = true
+hyper_v014.workspace = true
```

Comment:
We shouldn't need to add this dep -- if we need anything from hyper_v014, we should re-export those bits from ext/fetch.

Reply:
Just exported `Name` from `deno_fetch`. Should it be in a `dns` module or something, so that it indicate it relates to DNS?